<launch>
  <include file="$(find dynamixel_control)/launch/dynamixel_control.launch"/>
  <node pkg="simple_walk" type="simple_walk" name="simple_walk" output="screen">
    <param name="period" value="10.0"/>
    <param name="wheel_speed" value="0"/>
    <remap from="simple_walk/target_positions" to="dynamixel_control/setpositions"/>
    <remap from="simple_walk/target_speeds" to="dynamixel_control/setspeeds"/>
  </node>
</launch>

